src/pymortests/docker_ci_smoketest.py

```diff
@@ -12,3 +14,27 @@
                     reason='Guarantee only valid in the docker container')
 def test_config(pkg):
     assert getattr(config, f'HAVE_{pkg}')
+
+
+def test_no_dune_warnings():
+    _test_dune_import_warn()
+
+
+@pytest.mark.skipif(condition=not os.environ.get('DOCKER_PYMOR', False),
+                    reason='Guarantee only valid in the docker container')
+def test_dune_warnings(monkeypatch):
+    from dune import xt, gdt
+    monkeypatch.setattr(gdt, "__version__", "2020.0.0")
+    monkeypatch.setattr(xt, "__version__", "2020.0.0")
```

Comment:
Worth using the minimum of `version_ranges` from above and computing unsuitable versions (by `-1` or the like), to only have one occurrence of hard-coded versions?

Reply:
These versions don't actually exist and so are always older than any we create for those bindings. So I think it's ok not to make the version_ranges importable from the config module.